docs/examples/config.toml

```diff
@@ -1,108 +1,105 @@
 # ==============================================================================
-# Neurogym configuration template.
-# This template contains all the variables supported by the configuration module.
-# Please uncomment any variables that you would like to customise.
+# NeuroGym configuration template
+# This file defines all the customizable variables used by the configuration module.
+# Uncomment and set only the variables you wish to override.
+# ==============================================================================
+
+# ==============================================================================
+# General
+# ==============================================================================
+
+# Directory used to save output data (e.g., plots, logs, model checkpoints).
+# The default is <neurogym root>/local.
+# If you change this path, make sure it is outside your Git-controlled repository
+# or excluded via .gitignore if you want to avoid tracking output data.
+# local_dir = "/absolute/path/to/output"
+
+# ==============================================================================
+# Agent Configuration
+# NOTE: The agent configuration is currently only accessible via config instances.
+# The agent logic has not yet been integrated into the main code-base.
 # ==============================================================================
 
-# The local directory used for saving generated or downloaded data.
-# If you uncomment this option, please replace the placeholder with a valid path.
-# The default (<neurogym root>/local) is already in the .gitignore file,
-# and therefore excluded from commits. If you change this, make sure that
-# the custom path is either outside the repository tree or otherwise excluded
-# from git commits.
-# local_dir = <path-to-some-directory>
-
-# ==================================================
-# Agent configuration.
-# ==================================================
 [agent.training]
 
-# Number of trials during training.
-trials = 100
+# Metric used to trigger training (options: 'trials', 'steps').
```

Comment:
Are you thinking of using `trigger` to switch training on and off? My original intention was for `trigger` to act as a switch for outputting log messages or plots.

Reply:
I would decouple it for log messages and plots, since users might want to do both simultaneously.

For now, the trigger is used only to decide, given a value, whether it refers to the number of trials or steps to be used for the training. Since there is no real implementation of the config in the training this does not have any effect, but conceptually it makes sense.

Reply:
But it says "Metric used **_to trigger training_**". Would users use this variable to start training the agent? I don't think that's the intention, no?